doc/tutorials/basic/build_crossfilter_dashboard.md

```diff
@@ -0,0 +1,230 @@
+
+### Creating the Crossfiltering Plot
+
+The core of the dashboard is the `get_plots` function, which generates two interactive bar charts: one displaying the total capacity of wind turbines by year and another by manufacturer.
+
+```python
+@pn.cache
+def get_plots():
+    data = get_data()
+
+    # Define shared dataset
+    ds = hv.Dataset(data, ["t_manu", "p_year", "t_cap"], "t_cap")
+```
+
+The function starts by calling `get_data()` to retrieve the preprocessed dataset. It then uses HoloViews to define a dataset (`ds`) that encapsulates our data, specifying columns for manufacturers (`t_manu`), production year (`p_year`), and turbine capacity (`t_cap`).
+
+:::{note}
+
+In order for [HoloViews linked brushing](https://holoviews.org/user_guide/Linked_Brushing.html) (crossfiltering) to work the plots need to be generated from a shared dataset.
+
+:::
+
+#### Visualizing Data by Year and Manufacturer
+
+Next, we aggregate the data by year and manufacturer to create two separate plots. The plots are formatted to be responsive and use an accent color for consistency.
+
+```python
+    # Create plots
+    ds_by_year = ds.aggregate("p_year", function=np.sum).sort("p_year")
+    ds_by_manufacturer = ds.aggregate("t_manu", function=np.sum).sort(
+        "t_cap", reverse=True
+    )
+    formatter = NumeralTickFormatter(format="0,0")
+    plot_by_year = hv.Bars(
+        ds_by_year, ("p_year", "Year"), ("t_cap", "Capacity (GW)")
+    ).opts(
+        responsive=True,
+        xrotation=90,
+        yformatter=formatter,
+        color=ACCENT,
+        tools=["hover"],
+        active_tools=["box_select"],
+    )
+    plot_by_manufacturer = hv.Bars(
+        ds_by_manufacturer, ("t_manu", "Manufacturer"), ("t_cap", "Capacity (GW)")
+    ).opts(
+        responsive=True,
+        xrotation=90,
+        yformatter=formatter,
+        color=ACCENT,
+        tools=["hover"],
+        active_tools=["box_select"],
+    )
+
+    return (plot_by_year + plot_by_manufacturer).cols(1)
+```
+
+The `hv.Bars` method creates bar charts, which are then customized with options (`opts`) for responsiveness, rotation of the x-axis labels, formatting of the y-axis values, and color styling.
+
+#### Linking the Selections
+
+The crossfiltering capability is enabled by linking the selections across the two plots. This means that interacting with one plot (e.g., selecting a range of years) will dynamically filter the data in the other plot to match the selection criteria.
+
+```python
+crossfilter_plots = hv.link_selections(get_plots())
+```
+
+The `hv.link_selections` function wraps our layout of plots, enabling interactive filtering across them. This crossfilter mechanism is what makes our dashboard powerful for data exploration.
+
+:::{note}
+
+Please note we create a `crossfilter_plots` object for every user session because sessions cannot share `link_selections` objects.
```

Comment:
I noticed that if the crossfilter_plots is shared across session you can select individually in each session. But if you click the reset button in one session it resets all the plots. Don't know if this is as expected or a bug.

Reply:
Very bizarre bug!

Reply:
Turns out this was not a bug but a consequence of sharing the `linked_selections` instance across sessions. The fact that the selections weren't propagating to all sessions is the actual bug but not relevant in this tutorial.